index.html

```diff
@@ -241,6 +241,58 @@ <h2 class="about-title">About me</h2>
             </ul>
         </div>
     </section>
+    <section class="form-container">
+        <h3 class="form-heading">
+            I'm always interested in hearing about new projects, 
+            so if you'd like to chat please get in touch.
+        </h3>
+        <form action="https://formspree.io/f/xayvbrnw" method="post" class="form">
+            <ul class="form-inputs">
+                <li class="name-item">
+                    <input class="input" type="text" placeholder="Full name" name="full-name" maxlength="30" required>
+                    <div class="input-devider"></div>
+                </li>
+                <li class="email-item">
+                    <input class="input" type="email" placeholder="Email address" name="email" required>
+                    <div class="input-devider"></div>
+                </li>
+                <li class="comment-item">
+                    <textarea class="input-area" placeholder="Enter text here" name="comments" maxlength="500" required></textarea>
+                </li>
+            </ul>
+            <input class="submit-button" type="submit" value="Get in Touch">     
+        </form>
```

Comment:
- [ ] As the last reviewer said It would be better semantically if you remove input fields from `ul` and `li` and just put them inside `<form>` tags.

- [ ] Please add the right margins under the input fields and the line break and also Kindly remove the border around the `textarea` and try to stick as much as possible to the given Figma design.
- [ ] For social media icons below please center them right on the middle of the page and make the line break above them take the whole width of the page like the design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/159747946-3364574f-5380-46b6-8c08-3b771bad117a.png)  | ![image](https://user-images.githubusercontent.com/68796232/159747897-4dea690e-c847-4310-bcee-f4635fc55204.png)|

Reply:
kindly check my edits